client/repolist.c

```diff
@@ -654,25 +657,20 @@ TDNFRepoListFinalize(
                 dwError = TDNFConfigReplaceVars(pTdnf, &pRepo->pszName);
                 BAIL_ON_TDNF_ERROR(dwError);
             }
-
             if(pRepo->pszBaseUrl)
             {
                 dwError = TDNFConfigReplaceVars(pTdnf, &pRepo->pszBaseUrl);
                 BAIL_ON_TDNF_ERROR(dwError);
             }
-
             if(pRepo->pszMetaLink)
             {
                 dwError = TDNFConfigReplaceVars(pTdnf, &pRepo->pszMetaLink);
                 BAIL_ON_TDNF_ERROR(dwError);
```

Comment:
same here

Reply:
Not here. This is no `else if`, these lines will be executed in sequence. There may be `dwError` set in the first call to `TDNFConfigReplaceVars`, but this would get overwritten with the next one.